apps/range-voting/app/components/Panels/VotePanelContent.js

```diff
@@ -129,6 +131,29 @@ class VotePanelContent extends React.Component {
       this.setState({ remaining: 100 - total })
     }
   }
+
+  getVoterState = async () => {
+    const result = await this.props.app
+      .call('getVoterState', this.props.vote.voteId, this.props.user)
+      .toPromise()
+
+    // TODO: Bignumber.js vs >8.2 supports .sum function to initialize from a sum of bignumbers, replace when updating
+    const totalVotesCount = result.reduce(
+      (acc, vote) => acc.plus(vote),
+      new BigNumber(0)
+    )
+    //
+
+    const voteWeights = result.map(e =>
```

Comment:
This is going to represent their 'relative' votes not the actual token amounts they voted with is this the desired behavior?